packages/atomic/cypress/integration/facets/facet-utils.ts

```diff
@@ -48,27 +48,28 @@ export function facetValueShouldDisplayInBreadcrumb(
 }
 
 export function assertBasicFacetFunctionality(selector: string, field: string) {
+  cy.wait('@coveoAnalytics');
```

Comment:
the tests in this file using getAnalyticsAt were often flaky, because they batch click on elements then wait for the multiple UA calls, instead of clicking then waiting, then clicking, etc. Which is the normal behaviour and doesn't cause race conditions issues. 🤞